distribution/ddf-common/src/main/resources/etc/fipsToIso.properties

```diff
@@ -1,5 +1,5 @@
 # FIPS 10-4 to ISO 3166-1 alpha 3 country codes from http://www.opengeocode.org/
```

Comment:
Should we remove the reference to http://www.opengeocode.org/? This website doesn't even seem to work anymore.